services/cd-service/pkg/cmd/server.go

```diff
@@ -121,6 +121,10 @@ func RunServer() {
 			logger.FromContext(ctx).Fatal("config.parse.error", zap.Error(err))
 		}
 
+		if c.BootstrapMode && !c.DbWriteEslTableOnly {
```

Comment:
Write EslTableOnly can be false and the DB still be disabled. NO_DB & WriteEslTableOnly = false should work with bootstrap mode, right?

Reply:
Now the condition is rewritten such that it depends on the return value of `ShouldUseOtherTables`, which is what it eventually boils down to in the `GetEnvironmentsConfig`.